go/probe/cmd/hades/hades.go

```diff
@@ -6,9 +6,8 @@ import (
 )
 
 func main() {
-	var ok bool
-	var hq string
-	if hq, ok = os.LookupEnv("PRELUDE_API"); !ok {
+	hq := os.Getenv("PRELUDE_API")
```

Comment:
Pretty sure Go supports a fallback value when getting an env var

Reply:
lmk what that is - this is what I see as normal

Reply:
Fallback, meaning a default value when the variable is not set. In most languages it is something like: get(var, fallback)

Reply:
thx ha i know what a fallback is @privateducky 

im saying lmk the exact command in Go, because I didn't find one

Reply:
Might not exist then! Thought it did but not 100%